### DISCUSSION TITLE
Cell phone numbers

### DISCUSSION THREAD

**Discussion by AntoniosBarotsis** [bchavez/Bogus#432] [Q&A] [answered]
I haven't used Bogus before so apologies if I missed this but it seems like cell phone numbers are not accessible?

The JSON files have a `cell_phone` similar to `phone_number` but only the latter seems accessible (through `Bogus.DataSets.PhoneNumbers`).

Is this intended?

Is there some other way to generate random cell phone numbers?

Thanks in advance!

**Comment by ChristoWolf** [comment, marked as answer]
If you want to target a specific phone number schema, you can simply use any of the `Random.Replace...` methods - or even more explicitly [these](https://github.com/bchavez/Bogus/blob/e29f0205f6af6d736570646f4b08231eda6dba19/Source/Bogus/DataSets/PhoneNumbers.cs#L27), see e.g. [this example usage](https://github.com/bchavez/Bogus/blob/e29f0205f6af6d736570646f4b08231eda6dba19/Examples/GettingStarted/Program.cs#L37).
